packages/lib-classifier/src/components/Classifier/components/QuickTalk/QuickTalkContainer.js

```diff
@@ -101,24 +86,24 @@ class QuickTalkContainer extends React.Component {
       page: 1,
       sort: 'created_at',  // PFE used '-created_at' to sort in reverse order, and I have no idea why.
     }
-        
+
     talkClient.type('comments').get(query)
-      .then (comments =>{
-        this.setState({ comments })
-        
+      .then (allComments =>{
+        setComments(allComments)
+
         let author_ids = []
         let authors = {}
         let authorRoles = {}
 
-        author_ids = comments.map(comment => comment.user_id)
+        author_ids = allComments.map(comment => comment.user_id)
         author_ids = author_ids.filter((id, i) => author_ids.indexOf(id) === i)
-        
+
         apiClient.type('users').get({ id: author_ids })
           .then(users => {
             users.forEach(user => authors[user.id] = user)
-            this.setState({ authors })
+            setAuthors(authors)
           })
-        
+
         talkClient.type('roles')
```

Comment:
Are Talk roles different from Panoptes roles? You can use the new client to get roles. The `useProjectRoles` hook has an example.